### PR TITLE
New extension: Pika, beautify images quickly

### PR DESCRIPTION
**Pull request by rishimohan**
## Description

This PR adds extension for [Pika.style](https://pika.style), a web app to quickly turn RAW images into beautiful images. This extensions has two commands: 

1) allows opening filesystem images in Pika
2) allows opening a URL as screenshot in Pika

## Screencast

https://user-images.githubusercontent.com/14031295/180607472-3d965ecf-1167-44ea-85be-6c910ee2896a.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by danielsequeira** [comment]
Hi @rishimohan 👋 

Thank you for your contribution to our store.

I tested the extension and found a couple of issues:

1. If I enter the command "Open Image in Pika" and use the file picker, if I don't select any file and cancel it, the next time I open Raycast I'm presented with this screen:
pika 2022-08-03 at 12 11 10

2. In the "Open URL in Pika" command, if I trigger the action to open the URL without inserting one, I get redirected to the following URL: http://localhost:3000/?utm_source=Pika%20for%20Raycast(URL)&use=https:// and presented with this in the browser:
Screenshot 2022-08-03 at 12 11 45
I believe that if the URL input is empty, we shouldn't allow the action to be triggered.

File: extensions/pika/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pika",
+  "title": "Pika",
+  "description": "Beautify screenshots quickly using Pika.style",
```
Comment (danielsequeira) [inline extensions/pika/package.json:5]: I would refer to it as either `pika.style` or just `Pika`

File: extensions/pika/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "Design Tools",
+    "Web",
+    "Media"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "openFromURL",
+      "title": "Open URL in Pika",
+      "subtitle": "Pika.style",
```
Comment (danielsequeira) [inline extensions/pika/package.json:18]: Normally we use the extension title here
```suggestion
      "subtitle": "Pika",
```

File: extensions/pika/package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      "description": "Open website or Image URL in Pika",
+      "mode": "view"
+    },
+    {
+      "name": "openImage",
+      "title": "Open Image in Pika",
+      "subtitle": "Pika.style",
```
Comment (danielsequeira) [inline extensions/pika/package.json:25]: Normally we use the extension title here
```suggestion
      "subtitle": "Pika",
```

File: extensions/pika/src/openFromURL.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  };
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.OpenInBrowser
+            title="Open in Pika.style"
+            url={makeURL()}
+            onOpen={async () => await showHUD("Opening in Pika.style...")}
```
Comment (danielsequeira) [inline extensions/pika/src/openFromURL.jsx:25]: I would refer to it as either `pika.style` or just `Pika`

**Comment by rishimohan** [comment]
@danielsequeira Had missed those cases, fixed them now. Thanks for the review :)

**Comment by rishimohan** [comment]
Thanks @danielsequeira!

**Pull request merged**